src/Components/weather.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from "react";
 import AirQuality from "./airQuality";
-import "../index.css";
+//import "../index.css";
```

Comment:
Is this commented out?  Does it need to be removed?

Reply:
Oh actually if I have that in the code, it is not passing in unit test cases.

Reply:
But there's no problem with styling if I remove calling the index.css